### ISSUE TITLE
R session Aborted

### ISSUE THREAD

**Issue by chri002** [mlverse/torch#1191]
I have tried to follow the basic torch guide: Guess the correlation; but whenever I get to the point of running the R training, an error ‘R session aborted’ returns.

**Comment by sebffischer** [comment]
You should provide the code you are trying to run as it is otherwise hard to help you.

**Comment by chri002** [comment]
The tutorial, and code, that I followed is "guess the correlation" from "https: //torch.mlverse.org /start /guess_the_correlation /".
The error message is created when executing the following command:
fitted <- net %>%
  setup(
    loss = function(y_hat, y_true) nnf_mse_loss(y_hat, y_true$unsqueeze(2)),
    optimizer = optim_adam
  ) %>%
  fit(train_dl, epochs = 10, valid_data = test_dl)

The other commands seem to work, I can run an inference, build the model, move it between RAM and VRAM. But when I run the training it seems to collapse.
I hope it can be resolved.
